### ISSUE TITLE
Allow negated angle attribues to have players look away from a coordinate

### ISSUE THREAD

**Issue by RuedigerLP** [PGMDev/PGM#920]
for example something like this angle="!x,y,z"

**Comment by CoWinkKeyDinkInc** [comment]
So the idea is the players will look exactly the opposite (180 degrees) of this angle?

**Comment by Pablete1234** [comment]
I do assume the idea is that the normal vector of player direction is inverted, so invert both yaw and pitch (rotate both of them by 180 degrees)

I assume this is targeted at the spawn tag, although this never mentions it so i'm not sure, it'd be nice if @RuedigerLP could give more of an explanation.

My main concern with this is it looks like a super nieche thing that will be used by one single map and will add more code-burden than help
At the end of the day you can just define the pitch & yaw manually to point wherever you want

Is there another use-case for this?

**Comment by RuedigerLP** [comment]
Yes ^, well the specific use case would be for obs spawns if they are in the middle of a map and you want obs to be able to see the map better when they join the server. I could also imagine it being useful in ffa maps though.

**Comment by Pablete1234** [comment]
If obs spawn is in the middle of the map you don't really get much value out of making view away from center, since the distance will be small either way, pointing towards mid and away from mid will get you pretty much the same view regardless, plus-minus a the size of the region

Even for FFA i don't see any real use, you usually will want players to all look towards the center
